app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -255,6 +255,16 @@ def add_ep_merge_special_issue!
     update!(form_json: JSON.dump(form))
   end
 
+  def conditionally_determine_ep_merge
+    actor = OpenStruct.new({ flipper_id: user_uuid })
```

Comment:
Is this object essentially a more robust incarnation of `User.find(user_uuid)` that's resilient against Redis expiration?

Reply:
Correct